components/atom/tag/src/_settings.scss

```diff
@@ -31,9 +31,9 @@ $w-atom-tag-clickable: 32px !default;
 
 // closable
 $bgc-atom-tag-closable-icon: transparent !default;
-$c-atom-tag-closable-icon: transparent !default;
+$c-atom-tag-closable-icon: $c-system !default;
```

Comment:
```suggestion
$c-atom-tag-closable-icon: $c-gray !default;
```

Reply:
@turolopezsanabria you are a GitHub hacker 😎

Reply:
This was made in order to avoid a breaking change... But making the breaking change is a design system decision that makes sense.

Reply:
Initially the icon was transparent by default, check the gif: https://github.com/SUI-Components/sui-components/pull/1610

Reply:
The change in the PR that you comment broke the tag in the rest of the verticals, with this PR we solve that error.
If you need, you can overwrite the color variable from your theme.